CHANGELOG.md

```diff
@@ -12,33 +12,58 @@ upgrading `notebook`.
 Use `pip install pip --upgrade` to upgrade pip. Check pip version with
 `pip --version`.
 
-## v7.2
+## 7.3
```

Comment:
Worth adding a link to JupyterLab changelog and extension migration too as for 7.2?

Reply:
yes that makes sense :+1: